app/models/channel/facebook_page.rb

```diff
@@ -47,14 +47,14 @@ def subscribe
       ]
     )
   rescue => e
-    Rails.logger.debug "Rescued: #{e.inspect}"
+    Rails.logger.debug { "Rescued: #{e.inspect}" }
```

Comment:
Same comment here as above, do we need to report this to Sentry rather than debug logging? This wouldn't be available in the production logs, right?

Reply:
https://github.com/chatwoot/chatwoot/issues/2736